### DISCUSSION TITLE
Possible to use ESP32-S3-MINI-N8 instead of ESP32-S3-MINI-N4R2

### DISCUSSION THREAD

**Discussion by RoelandK** [revk/xESP32-Faikout#470] [Q&A]
Hi,
I want to order 20-30 boards from JLCPCB and sell them for mainland Europe, to help a bit.
The n4r2 seems not to be available from JLCPCB, can we use the N8 instead? I didnt see any specific reason why we need the 2mb of psram?

anyone who knows?

**Comment by revk** [comment]
The 2M RAM is not actually required.

It means a different build. I could add an N8 to the release.

**Comment by RoelandK** [comment]
Hah great thanks for the quick answer!

**Comment by revk** [comment]
OK issuing a beta now which includes an S3-MINI-N8 build.

**Reply by RoelandK** [reply]
thanks, one more question i cant seem to find it - if i produce the PCBs i need to remove your logo correct?

**Reply by revk** [reply]
Yes please, and company name.
